admin/class-admin-asset-seo-location.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * @package WPSEO|Admin
+ */
+
+/**
+ * Determines the location of an asset within the SEO plugin.
+ */
+final class WPSEO_Admin_Asset_SEO_Location implements WPSEO_Admin_Asset_Location {
+
+	/**
+	 * @var string
+	 */
+	protected $plugin_file;
+
+	/**
+	 * The plugin file to base the asset location upon.
+	 *
+	 * @param string $plugin_file The plugin file string.
+	 */
+	public function __construct( $plugin_file ) {
+		$this->plugin_file = $plugin_file;
+	}
+
+	/**
+	 * @param WPSEO_Admin_Asset $asset The asset to determine the URL for.
+	 * @param string            $type  The type of asset.
+	 *
+	 * @return string The URL of the asset file.
+	 */
+	public function get_url( WPSEO_Admin_Asset $asset, $type ) {
+		$path = $this->get_path( $asset, $type );
+		if ( empty( $path ) ) {
+			return '';
+		}
+
+		if ( 'development' !== YOAST_ENVIRONMENT && ! $asset->get_suffix() ) {
+			$plugin_path = plugin_dir_path( $this->plugin_file );
+			if ( ! file_exists( $plugin_path . $path ) ) {
+
+				// Give a notice to the user in the console (only once).
+				WPSEO_Utils::javascript_console_notification(
+					'Development Files',
+					sprintf(
+					/* translators: %1$s resolves to https://github.com/Yoast/wordpress-seo */
+						__( 'You are trying to load non-minified files, these are only available in our development package. Check out %1$s to see all the source files.', 'wordpress-seo' ),
```

Comment:
This sentence seems a bit off. Perhaps try one of the following:

`You are trying to load non-minified files. These are only available in our development package`

or

`You are trying to load non-minified files, which are only available in our development package`